extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -290,6 +290,20 @@ export class SubscriptionService {
       return this.timer_
         .timeoutPromise(timeout, subscriptionPlatform.getEntitlements())
         .then(entitlement => {
+          if (
+            entitlement.granted &&
```

Comment:
It looks like `entitlement` can be `null` here since the next line of code after this `if` takes care of it. I'd recommend moving that line above to avoid null access errors.

Reply:
Done.